### PR TITLE
Add `ChromatographyConfiguration` class and remove `FluidHandling` class

### PR DESCRIPTION
**Pull request by kheal**
This PR addresses https://github.com/microbiomedata/nmdc-schema/issues/1920.  For full discussion of issue and explanation of proposed changes see that issue.

Overview of changes this PR applies: 

Current schema:
Screenshot 2024-05-14 at 12 23 24 PM

Proposed changes:
Screenshot 2024-05-14 at 12 22 27 PM

Thank you @brynnz22, @corilo, @SamuelPurvine, and @turbomam for the input to this.

### CODE REVIEW

File: src/data/invalid/ChromatographicSeparationProcess-no_has_calibration_with_GC.yaml
Comment (kheal) [inline src/data/invalid/ChromatographicSeparationProcess-no_has_calibration_with_GC.yaml:1]: Deleted this file - there is no longer a `has_calbration` slot on `ChromatographicSeparationProcess` class.  That rule has moved to `MassSpectrometry` class.

File: src/data/valid/ChromatographicSeparationProcess-GC-has_calibration.yaml
Comment (kheal) [inline src/data/valid/ChromatographicSeparationProcess-GC-has_calibration.yaml:1]: This file was refactored as a ChromatographyConfiguration example (see ChromatographyConfiguration-GC.yaml)

File: src/data/valid/Database-gc.yaml
Comment (kheal) [inline src/data/valid/Database-gc.yaml:1]: This example was renamed and expanded on for clarity (see Database-mass_spectrometry_gc.yaml)

**Comment by turbomam** [comment]
I'm going to approve and merge this now. I will be experimenting with it over the next day or two. It's possible that I might want to suggest some changes at that point.

**Review by turbomam — APPROVED** [review]

**Pull request merged**